src/kili/entrypoints/queries/asset/__init__.py

```diff
@@ -514,6 +532,8 @@ def count_assets(
             label_honeypot_mark_lte: Returned assets should have a label whose honeypot is lower or equal to this number.
             issue_type: Returned assets should have issues of type `QUESTION` or `ISSUE`.
             issue_status: Returned assets should have issues of status `OPEN` or `SOLVED`.
+            external_id_strictly_in: Returned assets should have external ids that match exactly the ones in the list.
+            external_id_in: Returned assets should have external ids that partially match the ones in the list.
```

Comment:
You're right, I added more info

I let you suggest something if you still find it unclear

Reply:
Yeah it is very good now thanks! 
I think that the name "externalIdIn" in the graphQL resolver is bad. It would rather be externalIdMatch or externalIdLike (when we build the query, the name of the operator is "Like" and not "In", which makes more sens for what we want to achieve). But maybe we shouldn't change the name of the argument in python to stay consistent with the resolver ? Any strong opinion on this ?

Reply:
I think it's best to stay consistent with the resolver, and I find the externalIdIn and externalIdStrictlyIn to be explicit enough IMO